src/policy/interface.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_POLICY_INTERFACE_H
+#define BITCOIN_POLICY_INTERFACE_H
+
+#include <string>
+
+class CCoinsViewCache;
+class CScript;
+class CTransaction;
+
+/**
+ * \class CPolicy
+ * Interface class for non-consensus-critical policy logic, like whether or not
+ * a transaction should be relayed and/or included in blocks created.
+ */
+class CPolicy
+{
+public:
+    virtual ~CPolicy() {};
+    virtual bool ApproveScript(const CScript& scriptPubKey) const { return true; };
+    /**
+     * Check for standard transaction types
+     * @return True if all outputs (scriptPubKeys) use only standard transaction forms
+     */
+    virtual bool ApproveTx(const CTransaction& tx, std::string& reason) const { return true; };
```

Comment:
Initially it was a pure abstract class but if I remember correctly somebody complained about it, and it's more useful for alternative policies that don't want to restrict certain things for the base class to always return true in all Approve_ methods.
CPolicy is not the default policy, CStandardPolicy is. CPolicy is just the interface. It just happen to also implement the simplest policy possible: never reject anything on policy grounds.